src/appshell/qml/Preferences/internal/CommonAudioApiConfiguration.qml

```diff
@@ -64,7 +64,25 @@ Item {
                 apiModel.deviceSelected(newValue)
             }
         }
+        
+        ComboBoxWithTitle {
+            id: sampleRate
+
+            title: qsTrc("appshell", "Sample rate:")
+            columnWidth: root.columnWidth
+
+            currentIndex: indexOfValue(apiModel.sampleRate)
+            model: apiModel.sampleRateList
+
+            navigation.name: "SampleRateBox"
+            navigation.panel: root.navigation
+            navigation.row: root.navigationOrderStart + 1
```

Comment:
This combobox should probably be placed after "Buffer size:". And after that, the navigation row should be corrected:
navigation.row: root.navigationOrderStart **+ 2**